paasta_tools/tron_tools.py

```diff
@@ -255,6 +280,47 @@ def get_cleanup_action(self, default_paasta_cluster):
         action_dict['name'] = 'cleanup'
         return self._get_action_config(action_dict, default_paasta_cluster)
 
+    def check_max_runtime(self) -> Tuple[bool, str]:
+        max_runtime = self.get_max_runtime()
+        return check_timedelta(max_runtime, 'max_runtime')
+
+    def check_expected_runtime(self) -> Tuple[bool, str]:
+        expected_runtime = self.get_expected_runtime()
+        return check_timedelta(expected_runtime, 'expected_runtime')
+
+    def check_actions(self) -> Tuple[bool, List[str]]:
+        actions = self.get_actions('dummy-default')
```

Comment:
Why is it that we need a dummy-default here? How is it that the object itself doesn't have enough information to "know" what the config is in the first place?

Should the cluster we loaded he config from be part of the object? (self.cluster?) or can we get away with None?

Reply:
Ah, it totally can be None. Fixed.